rsocket/RSocketServer.cpp

```diff
@@ -160,6 +160,7 @@ void RSocketServer::onRSocketResume(
   auto result = serviceHandler->onResume(resumeParams.token);
   if (result.hasError()) {
     VLOG(3) << "Terminating RESUME attempt from client.  No ServerState found";
+    result.value()->rSocketStateMachine_->stats().resumeFailure();
```

Comment:
@somasun Also, do we want to identify this kind of failure in a different way? (another function? Or a variant of the other resume?)

Reply:
Should we provide the Stats object to RSocketServerHandler too?
Or we can just provide it to RSocketServer already.